src/lib/repositories/impl/product_repository_impl.py

```diff
@@ -10,19 +12,39 @@ def __init__(self):
 
     def add(self, product):
         product.id = self._current_id
+        product.create_date = datetime.now()
```

Comment:
where is `product.udpated_by = product.created_by` same comment for the dates?

Reply:
done.